databuilder/databuilder/extractor/es_base_extractor.py

```diff
@@ -48,7 +50,10 @@ def _get_index_mapping_properties(self, index: Dict) -> Optional[Dict]:
         mappings = index.get('mappings', dict())
 
         try:
-            properties = list(mappings.values())[0].get('properties', dict())
+            if self.es_major_version >= 7:
```

Comment:
would you mind extending tests to verify both cases?

Reply:
let me know how can i extend the test, please.  anyway, I already tested and using at v7.6 and v6.7.

Reply:
I would create method similar to this one (in the same class): https://github.com/amundsen-io/amundsen/blob/main/databuilder/tests/unit/extractor/test_es_metadata_extractor.py#L75 but mock the es client so it behaves like it's version 7 (and return payload expected by version 7) and make similar assertions